api/graphql/index.js

```diff
@@ -249,6 +254,7 @@ export function makeAuthenticatedQueries (userId, fetchOne, fetchMany) {
     person: (root, { id, email }) => fetchOne('Person', id || email, id ? 'id' : 'email'),
     post: (root, { id }) => fetchOne('Post', id),
     posts: (root, args) => fetchMany('Post', args),
+    responsibilities: (root, args) => Responsibility.fetchAll(args),
```

Comment:
why not use fetchMany as we do elsewhere? i think that handles the queryset stuff, though maybe we dont need that since there shouldnt be too many responsibilities